src/stcal/jump/twopoint_difference.py

```diff
@@ -152,186 +157,203 @@ def find_crs(
         for grp in range(dat.shape[1]):
             if np.all(np.bitwise_and(gdq[integ, grp, :, :], dnu_flag)):
                 num_flagged_grps += 1
-    total_groups = nints if only_use_ints and nints else nints * ngrps - num_flagged_grps
-    if (ngrps < minimum_groups and only_use_ints and nints < minimum_sigclip_groups) or (
-        not only_use_ints and nints * ngrps < minimum_sigclip_groups and ngrps < minimum_groups
-    ):
+    if only_use_ints:
+        total_sigclip_groups = nints
+    else:
+        total_sigclip_groups = nints * (ngrps - num_flagged_grps)
+    total_groups = nints * (ngrps - num_flagged_grps)
+    total_diffs = nints * (ngrps - 1 - num_flagged_grps)
+    total_usable_diffs = total_diffs - num_flagged_grps
+    if (ngrps < minimum_groups and only_use_ints and nints < minimum_sigclip_groups) or \
```

Comment:
For multiline conditionals, put the whole thing in parentheses.  This will also allow you to remove the slash at the end of the first line.

Reply:
I added the parentheses.